src/client/telemetry/index.ts

```diff
@@ -656,6 +656,10 @@ export interface IEventNamePropertyMapping {
     [Telemetry.UserInstalledPandas]: never | undefined;
     [Telemetry.UserDidNotInstallJupyter]: never | undefined;
     [Telemetry.UserDidNotInstallPandas]: never | undefined;
+    [Telemetry.PythonModuleInstal]: {
+        moduleName: string;
+        action: 'displayed' | 'installed' | 'ignored' | 'disabled' | 'failed';
```

Comment:
Generic way to track all module isntallation prompts, displayed & actions taken by user.